fedimint-derive/src/lib.rs

```diff
@@ -257,7 +263,12 @@ fn derive_enum_decode(ident: &Ident, variants: &Punctuated<Variant, Comma>) -> T
                     let total_bytes = bytes.len() as u64;
                     if read_bytes != total_bytes {
                         return Err(::fedimint_core::encoding::DecodeError::new_custom(anyhow::anyhow!(
-                            "Partial read: got {total_bytes} bytes but only read {read_bytes}"
+                            concat!(
+                                "Partial read: got {total_bytes} bytes but only read {read_bytes} when decoding ",
+                                stringify!(#ident),
+                                "::",
+                                stringify!(#variant)
+                            )
                         )));
```

Comment:
doesn't work as expected, anyhow! will not format the string

concat!(..) will be treated as `expr` not `literal`

Reply:
will fix tomorrow